### ISSUE TITLE
what is the demo 1 mean?

### ISSUE THREAD

**Issue by enjoyyi00** [aimerykong/Low-Rank-Bilinear-Pooling#1]
so you did not implement the Low-Rank-Bilinear-Pooling model in the paper(cvpr 2017) using Caffe?

so,i have i problem, what is the demo 1 mean?

**Comment by aimerykong** [comment]
No, there is no caffe version of the low-rank bilinear method. There are
several reasons in providing demo 1 --
1. Caffe runs a little faster than matconvnet, so simply applying PCA to do
dimensionality reduction and the bilinear training using caffe is a fast
combo to play with.
2. It seems combining our low-rank theory plus other kernel methods
achieves faster training and better performance.
